app/helpers/application_helper.rb

```diff
@@ -309,4 +313,13 @@ def get_class_for_any_path(paths)
     return ''
   end
 
+  def get_class_for_any_path_startswith(paths)
+    for path in paths do
+      if request.path.starts_with?(path)
+        return $active_css_class
+      end
+    end
+    return ''
+  end
+
```

Comment:
I would suggest refactoring this into something more rubyesque like
```
def get_class_for_any_path_startswith(paths)
  if paths.any? { |path| request.path.starts_with?(path) }
    return $active_css_class
  end
  return ''
end
```

Reply:
Yeah, this is similar to Python list comprehension, didn't know Ruby had that ;)